package.json

```diff
@@ -13,21 +13,25 @@
     "check-types": "tsc -p tsconfig.json --noEmit",
     "clean": "rm -rf node_modules && rm -rf build && yarn install",
     "codegen": "graphql-codegen --config codegen.ts",
-    "cy:open": "cypress open",
+    "cy:open": "yarn evg-db-ops --reseed-and-dump; cypress open",
     "cy:run": "cypress run",
     "cy:test": "cypress run --spec",
     "deploy:beta": "yarn build:beta && env-cmd -e beta yarn deploy:do-not-use",
     "deploy:staging": "yarn build:staging && env-cmd -e staging yarn deploy:do-not-use",
     "deploy:prod": "env-cmd -e production ts-node scripts/deploy/run-deploy",
     "deploy:do-not-use": ". ./scripts/deploy/deploy.sh",
     "dev": "env-cmd -e devLocal -r .env-cmdrc.local.json yarn start",
+    "dump-evg": "scripts/evg-db-ops.sh --dump",
     "diff-schema": "ts-node scripts/diff-local-schema-with-remote/index.ts",
     "eslint:fix": "yarn eslint:strict --fix",
     "eslint:staged": "STRICT=1 eslint",
     "eslint:strict": "STRICT=1 eslint '*.{js,ts,tsx}' 'src/**/*.ts?(x)' 'scripts/**/*.js' 'cypress/**/*.ts' 'src/gql/**/*.graphql'",
+    "evg-db-ops": "scripts/evg-db-ops.sh",
```

Comment:
I'd prefer to either have all custom yarn scripts for DB operations or use strictly `yarn evg-db-ops --option-flag`. Right now there's a mix of both being used